examples/00_quick_start/dkn_MIND_dataset.ipynb

```diff
@@ -21,21 +21,24 @@
     "- It makes use of knowledge entities and common sense in news content via joint learning from semantic-level and knnowledge-level representations of news articles.\n",
     "- DKN uses an attention module to dynamically calculate a user's aggregated historical representaition.\n",
     "\n",
-    "## Data format:\n",
+    "## Reference environment \n",
+    "we tested this notebook on two environment:\n",
+    "cpu with tensorflow==1.15.2 and gpu with tensorflow==1.15.2 at July 1, 2020.\n",
     "\n",
+    "## Data format:\n",
     "### DKN takes several files as input as follows:\n",
     "- training / validation / test files: each line in these files represents one instance, the format is : <br> \n",
     "`[label] [userid] [CandidateNews]%[impressionid] `<br> \n",
     "e.g., `1 train_U1 N1%0` <br> \n",
     "- user history file: each line in this file represents a users' click history, the format is : <br> \n",
     "the format is like: <br> \n",
-    "`[Userid] [clickedNews0:newsid] [clickedNews1:newsid] ...`\n",
-    "e.g., `train_U1 clickedNews0:N1 clickedNews1:N2` <br> \n",
+    "`[Userid] [newsid1,newsid2...]`<br>\n",
+    "e.g., `train_U1 N1,N2` <br> \n",
     "- document feature file:\n",
     "It contains the word and entity features of news. News article is represented by (aligned) title words and title entities. To take a quick example, a news title may be : Trump to deliver State of the Union address next week , then the title words value may be CandidateNews:34,45,334,23,12,987,3456,111,456,432 and the title entitie value may be: entity:45,0,0,0,0,0,0,0,0,0. Only the first value of entity vector is non-zero due to the word Trump.\n",
     "the format is like: <br> \n",
     "`[Newsid] [w1,w2,w3...] [e1,e2,e3...]`\n",
-    "- word embedding/entity embedding/ context embedding files: store word/ entity/ context embedding in .npy format.\n",
+    "- word embedding/entity embedding/ context embedding files: Each line in these files is an embedding, the k<sup>th</sup> line corresponding to the index k in document feature file. We used GloVe\\[4\\] vectors to initialize the word embedding. In this experiment, we train entity embeddings using TransE\\[2\\]. Context embedding is the average embedding of the entity's immediate neighbors in the knowledge graph. We store these embedding in .npy format.\n",
```

Comment:
Please polish this description with your best effort. It is terrible for now.  Unclear. Unordered.  If I am a paper reviewer, I won't read through your other sections and give a rejection directly.

Reply:
updated